packages/feedback/src/FeedbackForm.js

```diff
@@ -98,16 +100,20 @@ const FeedbackForm = ({
   ) : (
     <>
       <ModalHeader
-        aria-live="assertive"
+        aria-modal="false"
         id="feedback-form-header"
+        tag="h2"
+        tabIndex="0"
         {...modalHeaderProps}
       >
         {prompt || `Tell us what you think about ${name}`}
       </ModalHeader>
       <Form
         aria-label="Feedback Form"
         aria-describedby="feedback-form-header"
+        aria-labelledby="feedback-form-header"
```

Comment:
aria-label and aria-labelledby do the same thing, they just accomplish it in two ways. If the label is already available on the screen, aria-labelledby just points to the element. If the label is not available on screen then aria-label allows you to write it in. aria-labelledby will take precedence when multiple labels are assigned.